misc/deprecated/domain/builders/atrium-domain-builders-common/src/main/kotlin/ch/tutteli/atrium/domain/builders/utils/VarArgHelper.kt

```diff
@@ -37,18 +37,21 @@ interface VarArgHelper<out T> {
  * @throws IllegalArgumentException in case the iterable is empty.
  */
 inline fun <reified T> toVarArg(iterableLike: IterableLike): Pair<T, Array<out T>> =
+    iterableToPair(iterableLikeToIterable(iterableLike))
+
+inline fun <reified T> iterableLikeToIterable(iterableLike: IterableLike): Iterable<T> =
```

Comment:
please check if `inline` and `reified` are really required.

Reply:
if we remove `inline` and `reified` from `iterableLikeToIterable` we will get a warning in the mapping `Unchecked cast: Any? to T`, Should I remove `inline` and `reified` and suppress the warning?

Reply:
ah right forgot about that. no keep it in this case (and in the other places I said maybe we can remove it)